docs/semp.md

```diff
@@ -0,0 +1,416 @@
+# SE Management Plan (SEMP)
+
+### Metadata
+
+| Item | Description |
+| --- | --- |
+| Maintainer(s) | A.M. Smith |
+
+## Terms
+
+Terms | Descriptions
+--- | ---
+Core Team | Members of the [GitHub Team](https://github.com/orgs/Arrow-air/teams) relevant to product
+Release/Version | "Release" and "Version" are used interchangeably in this document
+
+## Overview
+
+The Systems Engineering Management Plan (SEMP) is a guide to managing technical
+processes and documentation.
+
+This document outlines:
+- Technical processes
+- Creation, modification, and review of technical
+documentation
+- Review processes
+
+## :busts_in_silhouette: Code Collaboration
+
+### :muscle: Contributing
+
+The process of contributing to a GitHub repository is well documented online.
+1. Make a copy of the repository on the local machine
+2. Initialize a "branch" using git
+    - A branch is a set of changes with a label, e.g. `readme-updates`
+3. Edit the relevant document(s)
```

Comment:
Might be worth noting that the branch name should also have the github username (e.g. `goodluckh/readme-updates`). This makes it easier for searching and filtering.

Reply:
https://github.com/Arrow-air/se-services/pull/1/commits/f1ce881223e9986116a69d481dd93e4c408d444c